src/pages/join/LogoutPage.tsx

```
@@ -13,14 +13,22 @@ const LogoutPage = () => {
         await axiosClient.post('/oauth/logout');
         logout();
         navigate('/');
-      } catch (error) {
-        console.error('로그아웃 실패:', error);
-        alert('로그아웃에 실패했습니다. 다시 시도해주세요.');
-        navigate('/menu');
+      } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)